apps/chat/src/components/Common/ItemContextMenu.tsx

```diff
@@ -257,7 +262,7 @@ export default function ItemContextMenu({
         name: t('Publish'),
         dataQa: 'publish',
         display:
-          !isEmptyConversation &&
+          (!isEmptyConversation || isPlayback) &&
```

Comment:
Convert changes into this file and follow this TODO-comment
https://github.com/epam/ai-dial-chat/pull/1436/files#diff-23c1c91e8e88c25dcb3a4e432eedbac27649675676ad81e21330bc4cebdb6279R633

Reply:
Converted and changed as suggested in this TODO-comment
https://github.com/epam/ai-dial-chat/pull/1436/files#diff-23c1c91e8e88c25dcb3a4e432eedbac27649675676ad81e21330bc4cebdb6279R633